Dockerfile.prow

```diff
@@ -2,16 +2,11 @@
 # and a workspace (GOPATH) configured at /go.
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
should we also use this image as the base

`FROM  registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux-amd64`

Reply:
the prow will automatically replace it for us, the Dockefile is more suitable for the local tests now...